pubspec.yaml

```diff
@@ -13,5 +13,5 @@ dev_dependencies:
   pedantic: ^1.9.2
 
 environment:
-  sdk: ">=2.8.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
```

Comment:
null safety is stable, you can remove `-0`
https://dart.dev/null-safety/migration-guide#step5-publish

same for the `example`

Reply:
Done.

Reply:
Unrelated to this PR but I see constraint  `flutter: ">=0.2.5 <2.0.0"` Considering that the current version is `2` I think it can cause an issue when there's the next stable update. Maybe it should be changed just for safety to something newest, like `flutter: ">=1.17.0"`?

Reply:
Specifying a upper constraint for flutter is deprecated and will be ignored. So this would still work with flutter 2.0 but should be removed because it will be confusing and pub will warn about it when publishing.
https://github.com/dart-lang/pub/pull/2780